python/pdstools/plots/plot_base.py

```diff
@@ -1126,8 +1348,59 @@ def plotTreeMap(
             log=log,
             midpoint=midpoint,
             format=format,
-            context_keys=self.context_keys,
+            context_keys=tree_levels,
             value_in_text=value_in_text,
             query=query,
             **kwargs,
         )
+
+    def plotPredictorCount(
+        self,
+        facets: str = "Configuration",
+        query: Union[str, dict] = None,
+        by: str = "Type",
```

Comment:
```
df.groupby(groupby_cols).
agg(pl.n_unique("PredictorName").over("Type").cast(pl.Int64).alias("Predictor Count"),
pl.n_unique("PredictorName").alias("overall"))
```
over can't be implemented in groupby. Try to find a way to avoid creating 2 dataframes